pkg/app/application.go

```diff
@@ -104,6 +105,10 @@ func (a *Application) loadCustomizedResMap() (resmap.ResMap, error) {
 	if err != nil {
 		errs.Append(errors.Wrap(err, "rawResources"))
 	}
+	err = crds.RegisterCRDs(a.loader, a.kustomization.CRDs)
+	if err != nil {
+		errs.Append(errors.Wrap(err, "CustomeResourceDefinitions"))
```

Comment:
i think this should be _RegisterCRDs_

The pattern i'm going for here is a java-like stack trace, so the error message almost looks like a call stack.

I'm just using method names since its easy.

It's fine to do something more creative than a method name, but if it's not clearly better then just stick to the method name.  That way golanf refactoring changes will keep it in sync (i think - they work with mentions in comments anyway).